python_coreml_stable_diffusion/torch2coreml.py

```diff
@@ -1301,11 +1350,19 @@ def parser_spec():
         ("The pre-trained model checkpoint and configuration to restore. "
          "For available versions: https://huggingface.co/models?search=stable-diffusion"
          ))
+    parser.add_argument(
+        "--refiner-version",
```

Comment:
Correct, this is meant to be used together with `--model-version`. There are a couple of ways to convert a refiner now, one is directly as the model in `--model-version`, which yes would convert an entirely different model naming convention and would require them to merge together manually. The other option is just run the same script they did for the base unet and add the `--refiner-version` param, which if all the other models were already converted would just add this refiner alongside the rest of them in the bundle. There is potential for this to get mixed up I think, but I also think there will not always be a tightly coupled refiner for each base model, so that makes it hard to know ad-hoc which refiner is meant to be bundled with which base - open to ideas on how to make that more clear. I'm planning on updating the README with this kind of info as well.